google-chat-bot/src/main/java/io/spine/chatbot/server/google/chat/ThreadChatRepository.java

```diff
@@ -63,7 +64,8 @@
     protected void setupEventRouting(EventRouting<ThreadId> routing) {
         super.setupEventRouting(routing);
         routing.route(BuildFailed.class, new RepositoryEventRoute<>())
-               .route(BuildRecovered.class, new RepositoryEventRoute<>());
+               .route(BuildRecovered.class, new RepositoryEventRoute<>())
```

Comment:
Here, "Repository" of Git clashes with "Repository" of Spine. Please rename this `EventRoute` class to make it more telling. My suggestion is `ByGitRepository`.